scripts/whitelists/package_whitelist.py

```diff
@@ -37,6 +37,7 @@
 batch_id  # unused variable (black_it/schedulers/base.py:77)
 batch_id  # unused variable (black_it/schedulers/round_robin.py:47)
 MABEpsilonGreedy  # unused class (black_it/schedulers/rl/agents/epsilon_greedy.py:26)
+seed  # unused variable (black_it/schedulers/rl/envs/base.py:59)
```

Comment:
for some reason this vulture error was activated after the first commit of this PR.